pxc/jenkins/qa_pxc_cross-pipeline.yml

```diff
@@ -0,0 +1,48 @@
+- job:
+    name: qa_pxc_cross-pipeline
+    project-type: pipeline
+    description: |
+        Do not edit this job through the web!
+    concurrent: true
+    disabled: false
+    node: micro-amazon
```

Comment:
I believe we can remove node part as we set labels for every step.

Reply:
Fixed !